test/client/transaction.test.ts

```diff
@@ -99,21 +110,21 @@ describe("TransactionMemoryStorage", () => {
             });
 
             it("properly transitions to failed state from pending", async function() {
-                const txn = txFactory(10, 11, 10101, 101, 0);
+                const txn = txFactory(10, 11, 1010, 101, 0);
```

Comment:
https://github.com/thehubbleproject/hubble-contracts/blob/tx2db/ts/decimal.ts#L69 is causing the issue so reduced the value. Unsure why it used to pass before.